racer_types/Animal.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
the (main) difference between dataclass and namedtuple is that namedtuple is immutable (after it's created, it can't be changed).
Why did you choose dataclass? do you expect to change the state of an animal after creating it? should it be possible?